README.md

```diff
@@ -527,7 +527,28 @@ The database name for the create script gets resolved in the following manner:
 1. Package name.
 > Note: 
 >- the generated script also uses the resolved database name via a setvar command.
->- if `IncludeCompositeObjects` is true, the composite objects (tables, etc.) from external references are also included in the generated script. This property defaults to `true`   
+>- if `IncludeCompositeObjects` is true, the composite objects (tables, etc.) from external references are also included in the generated script. This property defaults to `true`
+
+## Static code analysis
+Starting with version 2.6.2 of the SDK, there is support for running static code analysis during build. The SDK includes the following sets of rules:
```

Comment:
Maybe we'll want to ship this as 2.7.0 since this is a new feature?

Reply:
Fine by me - but is it breaking?

Reply:
I wouldn't go as far as to make this a 3.0 release no since it is not breaking as you have to opt-in to this feature anyway. But it is more than a simple patch to existing functionality, hence why I think 2.7.0 makes sense in this case.